lib/harvester.js

```diff
@@ -310,6 +309,40 @@ Harvester.prototype.after = function (name, fn) {
   return this;
 };
 
+Harvester.prototype.exportRoles = function () {
+    var roles = {};
+    _.forEach(this.createdResources, function (resource) {
+        _.forEach(resource.roles, function (role) {
+            roles[role] = roles[role] || [];
+            route.getRestMethods().forEach(function (httpMethod) {
+                if (resource[httpMethod]().options.notAllowed) {
+                        return;
+                    }
+                    var methodRoles = resource[httpMethod]().roles;
+                    var permission = resource[httpMethod]().getPermission();
```

Comment:
I'm thinking that you want to call this permissionName and .getPermissionName, since getPermission sounds like it returns a boolean that says yes or no.